api/server/service/jct/api.coffee

```diff
@@ -1231,9 +1290,13 @@ API.service.jct.sa = (journal, institution, funder, oa_permissions) ->
     if r.compliant isnt 'yes'
       # only check retention if the funder allows it - and only if there IS a funder
       # funder allows if their rights retention date is in the past
-      if journal and funder? and fndr = API.service.jct.funders funder
+      fndr = false
+      if funder?
+        fndrs = API.service.jct.suggest.funder funder
```

Comment:
I don't think this should prevent this PR moving forward, but I note that we are using the suggest endpoint to get the funder record, but since the API requires the funder id, you could just pull that record from the index by its id.